app-android/src/main/java/com/ustadmobile/port/android/view/ScopedGrantDetailFragment.kt

```diff
@@ -109,4 +122,48 @@ class ScopedGrantDetailFragment: UstadDetailFragment<ScopedGrantWithName>(
         }
 
     }
+}
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun ScopedGrantDetailScreen(
+    uiState: ScopedGrantDetailUiState,
+    onItemClick: (BitmaskFlag) -> Unit = {}
+){
+
+    Column (
+        modifier = Modifier
+            .fillMaxSize()
+    ){
+        uiState.bitmaskList.forEach { bitmask ->
+            ListItem(
+                modifier = Modifier.clickable {
+                    onItemClick(bitmask)
```

Comment:
The bitmask flag itself on detail mod should not be clickable. It won't go anywhere.